### ISSUE TITLE
Consider advising users not to use Trunk for wasm builds, or warn them about some caveats

### ISSUE THREAD

**Issue by rparrett** [bevy-cheatbook/bevy-cheatbook#253]
It seems like people love using trunk, but there are a number of ways using trunk can bite you. I totally agree with the decision to not give full blown instructions for a tool that you don't use / aren't familiar with. So I'm not totally sure what the right thing to do in the cheatbook is.

I've collected what I know about trunk (I am also not a trunk user) based on trying to help bevy users use it. I'll toss that information down below.

- Trunk's dev web server is meant for SPAs, but bevy's asset server needs the web server to serve `404`s for missing assets. By default when there is a missing file, trunk just serves `200` with the contents of `index.html` as the response body.
  
  It's a bit hard to follow what has been going on with trunk with regard to this issue. There are a number of old GH issues. There was a fork `trunk-ng` that added a `--no-spa` flag. That fork was seemingly "unforked", but I don't think that the option has made it into a release yet?

- The html file produces uses absolute paths to reference js/wasm files. This assume the project is being placed on its own server at the web root. That might be true in a lot of cases, but it creates a great deal of confusion when deploying to itch. On itch, your project is in a subdirectory and the CDN serves missing files as `403` errors.

  I am not sure if there is a workaround for this, other than manually editing the produced html to use `./` instead of `/`.

- The asset folder must be configured with [copy-dir](https://trunkrs.dev/assets/#copy-dir).

**Comment by janhohenheim** [comment]
Update by @MiniaczQ:
![image](https://github.com/bevy-cheatbook/bevy-cheatbook/assets/9047632/67581825-f8bd-4167-b535-55517d37d54f)
The first issue has been dealt with in the meantime 🎉

**Comment by MiniaczQ** [comment]
The `no_spa`:
https://github.com/trunk-rs/trunk/blob/c160ed3ff7c98a94ab00f14acbbd198268a92525/src/cmd/serve.rs#L42

**Comment by NiklasEi** [comment]
Since this issue came up on Discord discussing trunk usage with Bevy:
1. --no-spa [is supported](https://github.com/trunk-rs/trunk/blob/c160ed3ff7c98a94ab00f14acbbd198268a92525/src/config/rt/serve.rs#L38)
2. You can configure trunk for relative asset file paths now. Just add `public_url = "./"` to the `[build]` section of a `Trunk.toml` file (see [the bevy_game_template file](https://github.com/NiklasEi/bevy_game_template/blob/cb42cdf578d37f12c723856375d563c9a65818bf/Trunk.toml#L2))
